weather-app/helpers/geoData.js

```diff
@@ -0,0 +1,33 @@
+const request = require('request')
+
+const getGeoData = (location, callback) => {
+    const url = `https://api.mapbox.com/geocoding/v5/mapbox.places/${location}.json?access_token=pk.eyJ1IjoiZmxlbm9yeSIsImEiOiJjazY5M3g5dXIwYjB5M21sYXhjMmR0NTVsIn0.ycJkCu1iJ1wwxzKfLo3HOw`
+
+    request(url, (error, response) => {
+        if (error) {
+            callback(error, null)
+        }
+        
+        if (response === undefined) {
+            callback(error, null)
```

Comment:
Won't work - you don't have variable error at this point, you verified at line 7 that variable `error` exist or not